media-libs/openimageio/openimageio-1.7.16.ebuild

```diff
@@ -0,0 +1,122 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
+
+inherit cmake-utils python-single-r1 vcs-snapshot
+
+DESCRIPTION="A library for reading and writing images"
+HOMEPAGE="https://sites.google.com/site/openimageio/ https://github.com/OpenImageIO"
+SRC_URI="https://github.com/OpenImageIO/oiio/archive/Release-${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+
+X86_CPU_FEATURES=(
+	sse2:sse2 sse3:sse3 ssse3:ssse3 sse4_1:sse4.1 sse4_2:sse4.2
+	avx:avx avx2:avx2 avx512f:avx512f f16c:f16c
+)
+CPU_FEATURES=( ${X86_CPU_FEATURES[@]/#/cpu_flags_x86_} )
+
+IUSE="colorio doc ffmpeg field3d gif jpeg opencv opengl ptex python qt4 raw ssl +truetype ${CPU_FEATURES[@]%:*}"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RESTRICT="test" #431412
+
+RDEPEND=">=dev-libs/boost-1.62:=[python?,${PYTHON_USEDEP}]
+	dev-libs/pugixml:=
+	>=media-libs/ilmbase-2.2.0-r1:=
+	media-libs/libpng:0=
+	>=media-libs/libwebp-0.2.1:=
+	>=media-libs/openexr-2.2.0-r2:=
+	media-libs/tiff:0=
+	sys-libs/zlib:=
+	virtual/jpeg:0
+	colorio? ( >=media-libs/opencolorio-1.0.9:= )
+	ffmpeg? ( media-video/ffmpeg:= )
+	field3d? ( media-libs/Field3D )
+	gif? ( media-libs/giflib )
+	jpeg? ( virtual/jpeg:= )
+	opencv? (
+		>=media-libs/opencv-2.3:=
+		python? ( >=media-libs/opencv-2.4.8[python,${PYTHON_USEDEP}] )
+	)
+	opengl? (
+		virtual/glu
+		virtual/opengl
+	)
+	ptex? ( media-libs/ptex:= )
+	python? ( ${PYTHON_DEPS} )
+	qt4? (
+		dev-qt/qtcore:4
+		dev-qt/qtgui:4
+		dev-qt/qtopengl:4
+		media-libs/glew:=
+	)
+	raw? ( media-libs/libraw:= )
+	ssl? ( dev-libs/openssl:0= )
+	truetype? ( media-libs/freetype:2= )"
+DEPEND="${RDEPEND}
+	doc? ( app-doc/doxygen[latex] )"
+
+DOCS=( src/doc/${PN}.pdf )
+
+pkg_setup() {
+	use python && python-single-r1_pkg_setup
+}
+
+src_prepare() {
+	cmake-utils_src_prepare
+
+	use python && python_fix_shebang .
+}
+
+src_configure() {
+	# Build with SIMD support
+	local cpufeature
+	local mysimd=""
+	for cpufeature in "${CPU_FEATURES[@]}"; do
+		use ${cpufeature%:*} && mysimd+="${cpufeature#*:},"
+	done
+
+	# If no CPU SIMDs were used, completely disable them
+	[[ -z $mysimd ]] && mysimd="0"
+
+	local mycmakeargs=(
+		-DLIB_INSTALL_DIR="/usr/$(get_libdir)"
```

Comment:
True. I'll keep them for now. However, 1.6.18 is the last version of 1.6 branch, so I want to make it pretty, because it may be around for awhile as stuff still depends on 1.6.x.